android/gradle.properties

```diff
@@ -24,5 +24,8 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 
+org.gradle.jvmargs=-Xmx4096m
+
+
 # Version of flipper SDK to use with React Native
 FLIPPER_VERSION=0.99.0
```

Comment:
why this change ?

Reply:
Need more heap storage for gradle build so that's why added.